src/Compilers/CSharp/Portable/Symbols/Retargeting/RetargetingParameterSymbol.cs

```diff
@@ -23,6 +23,8 @@ internal abstract class RetargetingParameterSymbol : WrappedParameterSymbol
         /// </summary>
         private ImmutableArray<CSharpAttributeData> _lazyCustomAttributes;
 
+        private TypeWithAnnotations? _lazyTypeWithAnnotations;
```

Comment:
>private TypeWithAnnotations? _lazyTypeWithAnnotations; [](http://example.com/codeflow?start=8&length=54)

We use ```TypeWithAnnotations.Boxed``` to cache this information in symbols. There should be many examples

Reply:
➡️ This is a duplicate of the conversation in https://github.com/dotnet/roslyn/pull/69011#discussion_r1262022836, so we'll continue the discussion there.